packages/grafana-ui/src/components/Stack/Stack.mdx

```diff
@@ -28,7 +28,7 @@ Use the `Grid` component instead for these use cases:
 
```

Comment:
This doc is not yet clear to me.
People have 3 options: Stack, HorizontalStack, VerticalStack. So all three of them need to be mentioned and their differences require explanations in regards of component behaviour.
At the same time only HorizontalStack and VerticalStack are exported by `unstable.ts.` Does this mean that only these two are available? If yes, why is this doc about Stack?